firmware/controllers/settings.cpp

```diff
@@ -381,30 +381,6 @@ static void setOperationMode(int value) {
 	doPrintConfiguration();
 }
 
-static char pinNameBuffer[16];
-
-static void printThermistor(const char *msg, ThermistorConf *config, ThermistorMath *tm, bool useLinear) {
-	adc_channel_e adcChannel = config->adcChannel;
-	float voltage = getVoltageDivided("term", adcChannel PASS_ENGINE_PARAMETER_SUFFIX);
-	float r = getResistance(config, voltage);
-
-	float t = getTemperatureC(config, tm, useLinear);
-
-	thermistor_conf_s *tc = &config->config;
-
-	scheduleMsg(&logger, "%s volts=%.2f Celsius=%.2f sensorR=%.2f on channel %d", msg, voltage, t, r, adcChannel);
-	scheduleMsg(&logger, "@%s", getPinNameByAdcChannel(msg, adcChannel, pinNameBuffer));
```

Comment:
this functionality should not be removed

Reply:
this functionality is now all in TS